lib/Backend/GlobOptBailOut.cpp

```diff
@@ -482,6 +482,16 @@ GlobOpt::CaptureByteCodeSymUses(IR::Instr * instr)
 void
 GlobOpt::ProcessInlineeEnd(IR::Instr* instr)
 {
+    if (instr->m_func->hasArgLenAndConstOpt && instr->m_func->unoptableInlineArgCount == 0)
+    {
+        instr->m_func->hasUnoptimizedArgumentsAccess = false;
+        if (DoInlineArgsOpt(instr->m_func))
+        {
+            instr->m_func->m_hasInlineArgsOpt = true;
+            instr->m_func->frameInfo = instr->m_func->cachedInlineeFrameInfo;
```

Comment:
is it possible to have frameInfo non-null here? is it possible to have cacheInlineeFrameInfo null here? If either of those situations would be bad we should add an assert.

Reply:
This may be possible, would a check before entering this scope be better than an assert?

Reply:
12d2aa8e07c27501ef78db203fe04a8136392566